src/en/tutorial-client-basic.md

```diff
@@ -0,0 +1,367 @@
+Title: Basic client usage - tutorial
+TODO:  Warning: Ubuntu release versions hardcoded
+       Include links to planned tutorials (tutorial-bundles.md, tutorial-constraints.md)
+
+# Basic client usage - tutorial
+
+The goal of this tutorial is to give new Juju operators a solid introduction to
+the command line client. In so doing, several important concepts are
+introduced. It also discusses the use of command prefixes and aliases. Further
+reading suggestions are included at the end.
+
+## Prerequisites
+
+The following prerequisites are assumed as a starting point for this tutorial:
+
+ - You're using Ubuntu 18.04 LTS (Bionic).
+ - Juju (stable snap channel) is installed. See the [Installing Juju][install]
+   page.
+ - You have chosen a backing cloud and have created a controller for it. Refer
```

Comment:
I don't think we should introduce the term 'backing cloud'. I'd suggest just going with a 'cloud' or if we need to clarify the cloud in some way 'cloud for your model' or something along that line.